lib/models/application.js

```diff
@@ -41,6 +41,9 @@ const getApplicationModel = function (deps, opts) {
 
 	const deviceModel = once(() => require('./device').default(deps, opts));
 	const releaseModel = once(() => require('./release').default(deps, opts));
+	const inviteModel = once(() =>
```

Comment:
I think that the `once` here doesn't offer any benefit (just overhead), since are using the `inviteModel` in the same code block (while in other cases we use it inside method).